models/tags/Tags.ts

```diff
@@ -0,0 +1,4 @@
+export default interface Tags {
```

Comment:
Can you rename the `Tags` model to `Tag` to maintain consistency?

Reply:
Made the changes as per comments